doc/source/rllib/single-agent-episode.rst

```diff
@@ -0,0 +1,214 @@
+.. include:: /_includes/rllib/we_are_hiring.rst
+
+.. include:: /_includes/rllib/new_api_stack.rst
+
+.. include:: /_includes/rllib/new_api_stack_component.rst
+
+
+Episodes
+========
+
+RLlib stores and transports all trajectory data in the form of `Episodes`, in particular
+:py:class:`~ray.rllib.env.single_agent_episode.SingleAgentEpisode` for single-agent setups
+and :py:class:`~ray.rllib.env.multi_agent_episode.MultiAgentEpisode` for multi-agent setups.
+The data is translated from this `Episode` format to tensor batches (including a possible move to the GPU)
+only immediately before a neural network forward pass.
```

Comment:
Maybe we add here that this happens inside of the connector pipeline?

Reply:
done